README.md

```diff
@@ -1 +1,68 @@
-# rtk-system
\ No newline at end of file
+# rtk-system
+
+## Usage 
+Build a binary named rtk-system with:
```

Comment:
Start giving instructions and links on how to configure and use this module now, at this point, and if this repo doesn't have a license file, please add a license file to it.

Reply:
Was the license added?  Also a little bit more explanation ( you can crib from sparkfun and berry's language ) on what the use case is for.

Reply:
I added the license on main